lib/trento_web/controllers/v1/settings_controller.ex

```diff
@@ -40,4 +43,56 @@ defmodule TrentoWeb.V1.SettingsController do
 
     json(conn, %{})
   end
+
+  operation :get_api_key_settings,
+    summary: "Get Api key settings",
+    tags: ["Platform"],
+    responses: [
+      ok: {"Api Key settings", "application/json", Schema.Platform.ApiKeySettings},
+      not_found: Schema.NotFound.response()
+    ]
+
+  def get_api_key_settings(conn, _) do
+    with {:ok, api_key_settings} <- Settings.get_api_key_settings() do
+      settings_with_key =
+        Map.put(api_key_settings, :generated_api_key, generate_api_key!(api_key_settings))
```

Comment:
Wouldn't be simply better `api_key` field instead of `generated_api_key`?
I mean, it is pretty explicit that it is generated